examples/svelte-example/rollup.config.js

```diff
@@ -0,0 +1,91 @@
+import svelte from 'rollup-plugin-svelte';
+import resolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import livereload from 'rollup-plugin-livereload';
+import { terser } from 'rollup-plugin-terser';
+import sveltePreprocess from 'svelte-preprocess';
+import typescript from '@rollup/plugin-typescript';
+import injectProcessEnv from 'rollup-plugin-inject-process-env';
+
+const production = !process.env.ROLLUP_WATCH;
+
+function serve() {
+    let server;
+
+    function toExit() {
+        if (server) server.kill(0);
+    }
+
+    return {
+        writeBundle() {
+            if (server) return;
+            server = require('child_process').spawn('npm', ['run', 'start', '--', '--dev'], {
+                stdio: ['ignore', 'inherit', 'inherit'],
+                shell: true,
+            });
+
+            process.on('SIGTERM', toExit);
+            process.on('exit', toExit);
+        },
+    };
+}
+
+export default {
+    input: 'src/main.ts',
+    output: {
+        sourcemap: true,
+        format: 'iife',
+        name: 'app',
+        file: 'public/build/bundle.js',
+        globals: {
+            react: () => {},
+            'react-redux': () => {},
+        },
+    },
+    external: ['react', 'react-redux'],
```

Comment:
Had to do this to bail out of the dep inclusions... but this may not be right. Who knows how configs work? 🤦‍♂️ 💣